src/components/task/create-new/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useRef, useState } from "react";
+import propTypes from "prop-types";
+import { useDispatch } from "react-redux";
+import DOMPurify from "dompurify";
+import styles from "components/task/create-new/index.module.scss";
+import Button from "components/base/button";
+import TextArea from "components/base/text-area";
+import Image from "components/base/image";
+import { addTaskToTodo } from "store/actions/todo";
+import {
+  ALT_TAG_ICON_DELETE,
+  ATTRIBUTE_TITLE,
+  ERROR_MESSAGE_TASK_TITLE,
+  ICON_DELETE,
+  TYPE_BUTTON,
+} from "utils/const";
+import { generateTaskObject, parseForm } from "utils/helper";
+
+function CreateTask({ onSuccessfullTaskEntry }) {
+  const dispatch = useDispatch();
+  const taskInputRef = useRef(null);
+  const [errorMsg, setErrorMsg] = useState("");
+
+  function handleAddTaskSubmit(event) {
+    event.preventDefault();
+    const formData = parseForm(event.target);
```

Comment:
`event.target.value` can be used directly, why do we need to parse it as FormData?